.travis.yml

```diff
@@ -86,3 +92,12 @@ deploy:
       tags: true
       repo: plus3it/watchmaker
       condition: '"$TOXENV" == *"py27"*'
+  - provider: releases
+    draft: true
+    skip_cleanup: true
+    api_key:
+      secure: NZQua4PHfAugtez+Kkp+TZxxg8sgFPuTnEmJmQ/BBazUTWy5IGcMJFcQxI6y+YVmiiMshwV36B53sQaY9lLwZEaPARjPMNprBCRuGR436s5UrkmDSAl2uH7kbpEkbsIA2L4V++85v0q2PVSacgWW8abGs0mHydA6chHKewlvKiyI5PqlEhgABPx5LDIF2Wmrg/YfdPxtmMb9IHP0LN1sj5x59dxe727fbY6/kUpb4mrQ306EHBYSMi4E2mpP2zIlrt21KUuU8iFHNaRKUzSlq5Lva4JeoKlh3RT7G3YA4u+7dH5eJUxnHFFhFnLBIz1d2wDY1tVaEGX1CDAKW6lSCBBOarnLnDkvtLDD23NVUNXC81+7OLObaw63rtubfC3syZ1Kp517R4vgIZwRhBrvvJDr2IO5X/YtHVETBwkhDb8Ol8djXrwRbOeEgGWwkU66BbgX2HlcoggQqDhDWZWBosu7iz01pXvAFKiylNHbPULNV2gOM9HzY0kmxq3/zzwdBfLxJH8delSCK1uvQpZlmaBW6SA0/l7yuoxDN8KrXSRgwuE9A3XuMe6ZhbU5AZSqWA7vMdnwCg05xK2zo3Xohi7QE4AxY1jZW2ficBWWWFcdUTZpG+TRKC8LzQWBmZ61s9lNn+HfFC5yqYkJopLPeRYzfyeUJN79rr9leC04sVI=
+    on:
+      branch: master
+      repo: plus3it/watchmaker
+      condition: '"$TOXENV" == *"py27"*'
```

Comment:
Does this mean only if PY27 tests are passed? Do we have to be concerned about checking PY36 as well or are we just assuming all others are good because they had to have passed all of them via develop branch anyway?

Reply:
Right, so, travis executes the deploy for every _job_ in the test run, and we use a build matrix that results in 9 jobs. So, this condition ensures the deploy only executes once rather than for every job. Had to pick a job, py27 was as good as any. 🤷‍♂️